includes/Core/Assets/Assets.php

```diff
@@ -80,7 +80,8 @@ public function __construct( Context $context ) {
 	/**
 	 * Registers functionality through WordPress hooks.
 	 *
-	 * @since 1.0.0
+	 * @since 1.0.0 Function introduced.
```

Comment:
No need to add a dummy comment for the initial version of the method. Just the version number is enough:

```suggestion
	 * @since 1.0.0
```